cmd/dep/ensure_test.go

```diff
@@ -1,13 +1,16 @@
-// Copyright 2016 The Go Authors. All rights reserved.
+// Copyright 2017 The Go Authors. All rights reserved.
```

Comment:
nit: i think the standard thing here is to leave the original copyright date? honestly idk